core/iwasm/libraries/lib-wasi-threads/test/common.h

```diff
@@ -36,12 +36,11 @@ void
 run_long_task()
 {
     if (blocking_task_type == BLOCKING_TASK_BUSY_WAIT) {
-        for (int i = 0; i < TIMEOUT_SECONDS; i++)
+        for (;;)
             sleep(1);
```

Comment:
we can probably avoid the sleep and just have `for (;;) {};`

Reply:
Fixed, will heat CPU but not for long